fmriprep/workflows/bold.py

```diff
@@ -1285,16 +1285,11 @@ def _aslist(in_value):
         (inputnode, mask_mni_tfm, [('bold_mask', 'input_image')])
     ])
 
-<<<<<<< HEAD
-    bold_to_mni_transform = pe.Node(
-        MultiApplyTransforms(interpolation="LanczosWindowedSinc", float=True),
-        name='bold_to_mni_transform', mem_gb=8, n_procs=omp_nthreads)
-    # bold_to_mni_transform.terminal_output = 'file'
-=======
     bold_to_mni_transform = pe.Node(MultiApplyTransforms(
         interpolation="LanczosWindowedSinc", float=True, num_threads=omp_nthreads,
```

Comment:
This is re-adding num_threads to `bold_to_mni_transform.interface`.

Reply:
Sorry about that. Lazy merge with badly resolved conflicts.